servers/rendering/renderer_rd/renderer_scene_render_rd.cpp

```diff
@@ -1890,7 +1911,7 @@ void RendererSceneRenderRD::_render_buffers_post_process_and_tonemap(const Rende
 				// In stereo p_render_data->z_near and p_render_data->z_far can be offset for our combined frustrum
 				float z_near = p_render_data->view_projection[i].get_z_near();
 				float z_far = p_render_data->view_projection[i].get_z_far();
-				bokeh_dof->bokeh_dof_compute(buffers, camfx->dof_blur_far_enabled, camfx->dof_blur_far_distance, camfx->dof_blur_far_transition, camfx->dof_blur_near_enabled, camfx->dof_blur_near_distance, camfx->dof_blur_near_transition, bokeh_size, dof_blur_bokeh_shape, dof_blur_quality, dof_blur_use_jitter, z_near, z_far, p_render_data->cam_orthogonal);
+				bokeh_dof->bokeh_dof_compute(buffers, cam_attributes->dof_blur_far_enabled, cam_attributes->dof_blur_far_distance, cam_attributes->dof_blur_far_transition, cam_attributes->dof_blur_near_enabled, cam_attributes->dof_blur_near_distance, cam_attributes->dof_blur_near_transition, bokeh_size, dof_blur_bokeh_shape, dof_blur_quality, dof_blur_use_jitter, z_near, z_far, p_render_data->cam_orthogonal);
```

Comment:
Nitpicking but should we just give the cam_attributes pointer to bokeh_dof_compute?

Reply:
Done